iambic/plugins/v0_1_0/aws/organizations/scp/models.py

```diff
@@ -0,0 +1,582 @@
+    description: Optional[Union[str, list[Description]]] = Field(
+        "",
+        description="Description of the role",
+    )
+    type: ServiceControlPolicyItemType = Field(
+        default=ServiceControlPolicyItemType.SERVICE_CONTROL_POLICY
+    )
+    aws_managed: bool = Field(False)
+    policy_document: Union[PolicyDocument, List[PolicyDocument]]
+    targets: PolicyTargetProperties = Field(default=None)
+
+    tags: Optional[List[Tag]] = Field(
+        [],
+        description="List of tags attached to the role",
+    )
+
+    @property
+    def resource_type(self):
+        return "aws:iam:scp_policy:properties"
+
+    @property
+    def resource_id(self):
+        return self.policy_name
+
+    @validator("policy_document")
+    def sort_policy_document(cls, v: list[PolicyDocument]):
+        if not isinstance(v, list):
+            return v
+        sorted_v = sorted(v, key=lambda d: d.access_model_sort_weight())
+        return sorted_v
+
+    @classmethod
+    def sort_func(cls, attribute_name: str) -> Callable:
+        def _sort_func(obj):
+            return f"{getattr(obj, attribute_name)}!{obj.access_model_sort_weight()}"
+
+        return _sort_func
+
+    @validator("tags")
+    def sort_tags(cls, v: list[Tag]) -> list[Tag]:
+        sorted_v = sorted(v, key=cls.sort_func("key"))
+        return sorted_v
+
+
+class AwsScpPolicyTemplate(AWSTemplate, AccessModel):
+    template_type = AWS_SCP_POLICY_TEMPLATE
+    organization_account_needed: bool = Field(
+        True,
+        description="This template needs an organization account to be applied",
+    )
+    ARN_TEMPLATE = "arn:aws:organizations::{account_id}:policy/{organization_unit}/service_control_policy/{policy_id}"
+
+    properties: PolicyProperties = Field(
+        description="The properties of the scp policy",
+    )
+    account_id: str
+    org_id: str
+
+    def get_arn(self) -> str:
+        return self.ARN_TEMPLATE.format(
+            account_id=self.account_id if not self.properties.aws_managed else "aws",
+            organization_unit=self.org_id,
+            policy_id=self.properties.policy_id,
+        )
+
+    def _apply_resource_dict(self, aws_account: AWSAccount = None) -> dict:
+        resource_dict = super()._apply_resource_dict(aws_account)
+        resource_dict["Arn"] = self.get_arn()
+        return resource_dict
+
+    async def _apply_to_account(self, aws_account: AWSAccount) -> AccountChangeDetails:
+        if self.account_id != aws_account.account_id:
+            return AccountChangeDetails(
+                account=str(aws_account),
+                resource_id=self.properties.resource_id,
+                resource_type=self.resource_type,
+                new_value={},
+                current_value={},
+                proposed_changes=[],
+                exceptions_seen=[],
+            )  # type: ignore
+
+        client = await aws_account.get_boto3_client("organizations")
```

Comment:
Do we need a client in the org region? (I'm actually not sure)

Reply:
organizations are global, because of that I didn't added the region.